weis/glue_code/glue_code.py

```diff
@@ -631,7 +631,7 @@ def setup(self):
             # Connections to turbine constraints
             if modeling_options['WISDEM']['TowerSE']['flag']:
                 self.connect('configuration.rotor_orientation', 'tcons_post.rotor_orientation')
-                self.connect('rlds_post.tip_pos.tip_deflection',     'tcons_post.tip_deflection')
+                self.connect('aeroelastic.max_TipDxc',          'tcons_post.tip_deflection')
```

Comment:
I'm just confused by this, since earlier today I did merge the PR into weis develop...

Reply:
Okay, this is fixed. I had cherry picked your commits before the PR was merged and that was causing confusing